### DISCUSSION TITLE
功能栏空空如也

### DISCUSSION THREAD

**Discussion by Exusiai0627** [the1812/Bilibili-Evolved#3837] [提问 / Q&A]
所有软件已更新到最新版本
<img width="244" alt="截屏2022-11-23 下午1 03 16" src="https://user-images.githubusercontent.com/82081432/203472425-7f394d49-4ba3-4dcd-9916-100e1cdb61fd.png">

组件里也什么都没有，

<img width="445" alt="截屏2022-11-23 下午1 10 04" src="https://user-images.githubusercontent.com/82081432/203473049-f3ba4071-f4ea-4582-940e-41c2a4b5675e.png">

反复下载过好几次也没用

使用版本为：
macOS 10.15.7 
Chrome 107.0.5304.110
tampermonkey 4.18.0
bilibili Evolved 2.5.2

**Comment by the1812** [comment]
不看说明是吧
![image](https://user-images.githubusercontent.com/26504152/203472805-81ac604e-70aa-44bc-a1c5-351513e68801.jpeg)
